ci/taskcluster/bin/decision.py

```diff
@@ -7,15 +7,23 @@
 
 tasks = [
     {
+        'name': 'recipe-client-addon:build',
+        'description': 'Build Firefox with recipe-client-addon',
+        'command': 'normandy/recipe-client-addon/bin/tc/build.sh',
+    },
+    {
         'name': 'recipe-client-addon:test',
         'description': 'Test recipe-client-addon with gecko-dev',
         'command': 'normandy/recipe-client-addon/bin/tc/test.sh',
+        'dependencies': ['recipe-client-addon:build'],
+        'artifacts_from': {
```

Comment:
I'm not sure about this name. I started with `artifacts`, but it didn't sit right. To me `artifacts` would mean the artifacts this task exports. The actual intent of this is to declare what artifacts need to be imported from other tasks, and bind them to convenient names.

Making this a list with `envvar` keys feels good. What do you think about this?

```python
{
    'name': 'recipe-client-addon:test',
    ...
    'artifacts_from': [
        {
            'task_name': 'recipe-client-addon:build',
            'path': 'build.tar.gz',
            'envvar': 'BUILD_RESULT',
        },
    ],
}
```

`artifacts_from` is still a weird name, but I like it better than `artifacts`. I'd be happy to switch to something more apt.